readthedocs/core/notifications.py

```diff
@@ -35,7 +35,7 @@
                 Feel free to <a href="https://{{ PRODUCTION_DOMAIN }}/support/">report any feedback</a> you may have.
                 {% else %}
                 Our new <strong>beta dashboard</strong> is now available for testing.
-                <a href="https://beta.{{ PRODUCTION_DOMAIN }}/">Give it a try</a> and send us feedback.
+                <a href="https://app.{{ PRODUCTION_DOMAIN }}/">Give it a try</a> and send us feedback.
```

Comment:
Isn't the value of `PRODUCTION_DOMAIN` `app.readthedocs.org` already?

Reply:
No, we haven't done that change yet.

Reply:
But I mean in the beta instance. We don't need to add `app` here, otherwise it will be `app.app` when we do the change in all instances.

Reply:
The beta instance doesn't show this notification. When we make the switch, we will remove this notification, so there won't be duplicated `app.app` at that point.